src/testFixtures/java/org/opensearch/knn/KNNRestTestCase.java

```diff
@@ -852,7 +853,12 @@ protected Request constructScriptScoreContextSearchRequest(
         builder.endObject();
         builder.endObject();
         builder.endObject();
-        Request request = new Request("POST", "/" + indexName + "/_search");
+        String searchParamsString = "";
```

Comment:
is it possible to use UriBuilder to build URI instead of building string which is error prone?

Reply:
Updated to use UriBuilder